src/components/verify-tab/VerifyTabPage.vue

```diff
@@ -1,13 +1,62 @@
 <template>
   <div
-      class="tab-page verify-tab-page"
-      v-on:dragover="fileDragOver"
-      v-on:dragleave="fileDragLeave"
-      v-on:drop="fileDropped"
+    class="tab-page verify-tab-page"
```

Comment:
2 more spaces

Reply:
wait why?

Reply:
Because other files do that.

Reply:
e.g. that one https://github.com/lineageos-infra/updater-client/blob/main/src/components/changes-tab/ChangesGroup.vue

Reply:
Hmm interesting codestyle... ok